src/command/cmd_funcs.c

```diff
@@ -904,6 +904,28 @@ _account_set_auth(char* account_name, char* policy)
     return TRUE;
 }
 
+gboolean
+_account_set_max_sessions(char* account_name, char* max_sessions_raw)
+{
+    int max_sessions;
+    char* err_msg = NULL;
+    gboolean res = strtoi_range(max_sessions_raw, &max_sessions, -1, 100000, &err_msg);
```

Comment:
```suggestion
    gboolean res = strtoi_range(max_sessions_raw, &max_sessions, 0, INT_MAX, &err_msg);
```
Why `-1` as lower boundary?

Also this just made me realize that this should've been an `unsigned int` internally and not a signed one... but we can keep it like that.

@jubalh maybe we should think of introducing usage of `g_key_file_[gs]et_uint64()` for such values!?

Reply:
Sounds like a good idea! Also the `__attribute__` that you mentioned in another comment is a good idea!

Reply:
> Why `-1` as lower boundary?
> 
> Also this just made me realize that this should've been an `unsigned int` internally and not a signed one... but we can keep it like that for now.
> 
> @jubalh maybe we should think of introducing usage of `g_key_file_[gs]et_uint64()` for such values!?

`-1` looked logical to set it as "turned off" as in some programs for certain parameters. `100000` sessions sounded as logical limitation because I can't think of a scenario when someone would want to set more sessions. And the idea was to show for user that sets it to`9999999999` that it's possible to turn off by setting it to `-1` or `0`.

Reply:
Interesting interpretation, but `0` already turns the feature off!? Two "off" values doesn't make sense.

I would've expected that `-1` could mean "enable monitoring, but only report new devices" or something like that.

Please change the range to `0, INT_MAX` and we're fine to go with this PR.

Reply:
and btw. I also can't think of a use-case where `100000` sessions aren't enough, but I don't like the idea of just choosing value X of a limitation when there's only a technical limitation (`INT_MAX` in our case) in theory.

Fun fact: by using `100000` you were breaking a(nother very theoretical) case where someone wants to compile profanity for a 16bit CPU, since it's possible on such an architecture that an `int` is only able to hold values from `-32768 to 32767` :)